src/Nethermind/Nethermind.Optimism/Rpc/RegisterOptimismRpcModules.cs

```diff
@@ -54,6 +54,11 @@ protected override void RegisterEthRpcModule(IRpcModuleProvider rpcModuleProvide
             _logger.Warn($"SequencerUrl is not set. Nethermind will behave as a Sequencer");
         }
 
+        if (_config.SequencerUrl is not null)
+        {
+            _api.TxPool.Disable();
+        }
+
```

Comment:
Maybe better not to create it in the first place or create some NullTxPool?

Reply:
There is already `InitializeBlockchainOptimism.cs` so it is easy to override there

Reply:
We do have a `NullTxPool`